Update README.md (#39)
satabin/swam: README.md

## Getting Started

> To build it, we use [mill](http://www.lihaoyi.com/mill/). Mill is not available on on Unix boxes, but you can you use curl to install it (see Installation instructions in the [doc](http://www.lihaoyi.com/mill/)):

Reviewer: ```suggestion
To build it, we use [mill](http://www.lihaoyi.com/mill/). Mill is not packaged all Linux distributions, but you can you use curl to install it (see Installation instructions in the [doc](http://www.lihaoyi.com/mill/)):
```

Technically, it is available on some Unixes, e.g. MacOS through brew and Arch Linux in AUR, and FreeBSD has a port. Main lack is for Debian based and Redhat based distros :)

>     sh -c '(echo "#!/usr/bin/env sh" && curl -L https://github.com/lihaoyi/mill/releases/download/0.3.6/0.3.6) > $HOME/bin/mill && chmod +x $HOME/bin/mill'

Reviewer: I will have to think about upgrading this line when I upgrade mill :(

Author: agree. Unfortunately, github does not support `https://github.com/lihaoyi/mill/releases/download/latest/latest`

Reviewer: I actually doubt this makes sense here, because, it is simply copy/paste of what is in the mill documentation.

Author: I like self-contained getting started without opening any external URL.

Do you want me to remove this?

Reviewer: Well, it is not really, as OSX, Arch Linux, and FreeBSD users don't have the preferred way of doing it here, so this looks like this is the only way to isntall mill (which it is not). I would remove it. Your readme can never be self contained, unless it integrates the documentation for every used tool. And I don't want to do that.

Author: no need to further discuss. removed.

I you want to run the specification tests, just type:

>     $ mill runtime.test

Reviewer: I didn't realized you used that syntax, can you use the fenced code block instead please (for consistency across all markdwon files)

Author: done
